tests/browser/test/test_setup.js

```diff
@@ -339,8 +330,8 @@ async function dragBlockTypeFromFlyout(browser, categoryName, type, x, y) {
  */
 async function contextMenuSelect(browser, block, itemText) {
   await block.click({button: 2});
-  await browser.pause(200);
   const item = await browser.$(`div=${itemText}`);
+  await item.waitForExist();
```

Comment:
I think that note is referring to "implicit waiting" as described here: https://webdriver.io/docs/autowait/ where there is a setting you can enable to have webdriver automatically wait a given number of ms whenever a single element is searched for to see if it will exist within that time frame.

They recommend against implicit waiting, and the default setting for the timeout is 0, meaning there is no implicit waiting in our tests. If we don't explicitly wait when needed then a test would fail automatically if the element doesn't exist. So I think we do need some form of explicit wait here; in the old code that is a static number, and in the new code it just waits up to 500 ms (the default timeout value) for the element to exist.